packages/desktop/views/dashboard/TopNavigation.svelte

```diff
@@ -79,15 +80,19 @@
 
     <div class="right-button flex justify-end">
         {#if features?.network?.config?.enabled}
-            <button class="bg-white rounded-xl text-14 px-2" on:click={onNetworkClick}>Shimmer</button>
+            <button class="network-button" on:click={onNetworkClick}>
+                <NetworkIcon height={16} width={16} networkId={$activeProfile.network.id} />
+                <Text fontWeight={FontWeight.semibold} color="gray-800" darkColor="white"
+                    >{$activeProfile.network.name}</Text
+                >
+            </button>
```

Comment:
A few comments about this component:
- We should make this into a separate component 
- The designs have changed slightly, using a smaller border radius and a slightly smaller icon: 
    ![Screenshot from 2023-04-24 11-33-53](https://user-images.githubusercontent.com/44885822/234060004-b80f9a00-e9f2-4481-954b-7adb0a5f5b36.png)
- There's a white outline of the icon when hovering over it
- The hover effect in general seems a bit too dark, but we can also polish this later